pyproject.toml

```diff
@@ -40,7 +40,7 @@ dependencies = [
   "sciline>=24.06.0",
   "scipp>=23.8.0",
   "scippneutron>=23.9.0",
-  "scippnexus>=23.12.1",  # 23.12.0 and below deadlock in threaded use
+  "scippnexus>=24.6.0",  # Below 24.6.0 incompatible with Numpy 2.0
```

Comment:
Since ESSsans does not require Numpy 2 we should *not* have such a lower bound, I think? That is, put back 23.12.1.